controllers/rhmi/bootstrapReconciler.go

```diff
@@ -762,6 +770,43 @@ func (r *Reconciler) processQuota(installation *rhmiv1alpha1.RHMI, namespace str
 	return nil
 }
 
+func (r *Reconciler) reconcileCustomSMTP(ctx context.Context, serverClient k8sclient.Client) (rhmiv1alpha1.StatusPhase, error) {
+
+	smtp, err := cs.GetCustomAddonValues(serverClient, r.installation.Namespace)
+	if err != nil {
+		return integreatlyv1alpha1.PhaseFailed, err
+	}
+
+	validation := cs.ParameterValidation(smtp)
+	if validation == cs.Valid {
+		phase, err := cs.CreateOrUpdateCustomSMTPSecret(ctx, serverClient, smtp, r.installation.Namespace)
+		if err != nil {
+			return phase, err
+		}
+		r.installation.Status.CustomSmtp = &rhmiv1alpha1.CustomSmtpStatus{}
+		r.installation.Status.CustomSmtp.Active = true
+
+	} else if validation == cs.Partial {
```

Comment:
It seems that Valid Partial and Blank are mutually exclusive so you don't need the else here or below.